src/main/java/io/kestra/plugin/redis/Delete.java

```diff
@@ -0,0 +1,83 @@
+package io.kestra.plugin.redis;
+
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.annotations.PluginProperty;
+import io.kestra.core.models.executions.metrics.Counter;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.redis.services.RedisService;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import javax.validation.constraints.NotNull;
+import java.util.List;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Delete one or more keys."
+)
+@Plugin(
+    examples = {
+        @Example(
+            code = {
+                "type: io.kestra.plugin.redis.Delete",
+                "uri: redis://:redis@localhost:6379/0",
+                "keys:",
+                "   - keyDelete1",
+                "   - keyDelete2"
+            }
+        )
+    }
+)
+public class Delete extends AbstractRedisConnection implements RunnableTask<Delete.Output> {
+
+    @Schema(
+        title = "Redis keys",
+        description = "The list of redis keys you want to delete"
+    )
+    @NotNull
+    @PluginProperty(dynamic = true)
+    private List<String> keys;
+
+    @Schema(
+        title = "failedOnMissing",
+        description = "If some keys are not deleted, failed the task"
+    )
+    @Builder.Default
+    private Boolean failedOnMissing = false;
+
+
+    @Override
+    public Output run(RunContext runContext) throws Exception {
+        RedisService connection = this.redisFactory(runContext);
+        Long count;
+        boolean AllKeyDeleted;
+
+        count = connection.del(runContext.render(keys));
+        AllKeyDeleted = count == keys.size();
+
+        if (!AllKeyDeleted && failedOnMissing) {
+            connection.close();
+            throw new NullPointerException("Missing keys, only " + count + " key deleted");
+        }
+        connection.close();
+        runContext.metric(Counter.of("keyProcessed", count));
```

Comment:
```suggestion
        runContext.metric(Counter.of("keys.deleted", count));
```
By convention, no camel case in metrics, it's better from prometheus for example